src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -173,27 +173,51 @@
    constants/optimism-chain-id :optimism
    constants/arbitrum-chain-id :arbitrum})
 
+(defn calculate-fiat-change
+  "Returns unsigned fiat change"
+  [fiat-value change-pct]
+  (-> (if (number? change-pct) change-pct 0)
+      money/bignumber
+      (money/div 100)
+      (money/mul fiat-value)
+      money/absolute-value))
+
+(defn prettify-percentage-change
+  "Returns unsigned precentage"
+  [percentage]
+  (-> (if (number? percentage) percentage 0)
+      money/bignumber
+      money/absolute-value
+      (money/to-fixed 2)))
```

Comment:
I'm not sure about this `if` since the logic is not related to the function's purpose.

Why do we want to keep these kinds of checks? I mean, is someone calling this function not using a number and willing to get a `"$0.00"` as a result? and if so, maybe we shouldn't have initially mixed non-number values with numbers.

Reply:
The idea is to prevent the method from crashing on unexpected values such as `nil` or `strings` (token market values sometimes give out such values in rare cases) and to provide a uniform result `0.00`.